linguatec_lexicon_frontend/urls.py

```diff
@@ -15,6 +16,7 @@
     path('proyecto-linguatec/', views.LinguatecProjectView.as_view(), name='linguatec-project'),
     path('search/', views.SearchView.as_view(), name='search'),
     path('words/<int:pk>/', views.WordDetailView.as_view(), name='word-detail'),
+    path('admin/', admin.site.urls, name='index'),
```

Comment:
Maybe you are rigth, but this warning does not happen when I run the project/server, so I am not sure, maybe we should ask Santiago about this

Reply:
As @dunetna says, `admin` URL is commonly included on the project URLs (not on the app URLs). So please move it, thanks!

Reply:
Okey, done, this error was because probably i had a concept problem, but know I think I understand why it is done like that, thank you both for pointing out this error